src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -43,15 +44,20 @@ class IonRawBinaryWriter_1_1 internal constructor(
     private data class ContainerInfo(
         var type: ContainerType? = null,
         var isDelimited: Boolean = false,
+        var usesFlexSym: Boolean = false,
         var position: Long = -1,
         var length: Long = 0,
         // TODO: Test if performance is better with an Object Reference or an index into the PatchPoint queue.
         var patchPoint: PatchPoint? = null,
     ) {
-        fun clear() {
-            type = null
-            isDelimited = false
-            position = -1
+        /**
+         * Clears this [ContainerInfo] of old data and initializes it with the given new data.
+         */
+        fun reset(type: ContainerType? = null, position: Long = -1, isDelimited: Boolean = false) {
```

Comment:
The `clear()` method was useful, but it set all the fields to the default values, and then we had to set them to the correct values. The new `reset()` method skips some of the unnecessary field assignments.